server/protocols/jmap-rfc-8621/doc/specs/spec/mail/delegation.mdown

```diff
@@ -0,0 +1,162 @@
+# Delegations extension
+
+> :information_source:
+> Not Implemented
+
+## Context
+This extension is specific to the Apache James server which defines methods to manage users' JMAP delegation settings.
+
+This extension to the JMAP specification enables to:
+
+- Manage accounts that a user delegates to access his account.
+- Manage accounts that a user is delegated to access.
+
+## Addition capability
+The URN `urn:apache:james:params:jmap:delegation` represents support for the Delegation data type and associated API methods.
+
+Servers supporting this specification MUST add this property to the capabilities object.
+
+## Addition to the Session object
+Servers supporting this specification MUST add the delegated accounts to the `accounts` property of the Session object.
+
+## Behavioral changes
+When `urn:apache:james:params:jmap:delegation` is specified, access to a delegated account is enabled through standard JMAP API calls
+but MUST except API calls to manage delegated account's delegation setting to avoid privilege escalation.
+
+## Delegation methods
+A **Delegation** object stores information about an email address that either has access to the user's account or delegates access
+to the user. The semantics for this object depends on the context of usage methods: either manage delegate accounts or delegated accounts.
+
+The delegation object MUST have the following properties:
+- **id**: `Id` (immutable; server-set)
+  The id of the Delegation.
+- **username**: `String` (immutable)
+  The email address of the user that either delegates or is delegated to this user account.
+
+A user can have multiple delegate accounts as well as multiple delegated accounts.
```

Comment:
I find this sentence confusing... What about:

`A user can delegate his account to multiple users as well as have access to the account of multiple users.` 

Or something like that?